code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -35,7 +35,7 @@
 	desc = "A backpack that opens into a localized pocket of Blue Space."
 	origin_tech = "bluespace=4"
 	icon_state = "holdingpack"
-	max_w_class = 5
+	max_w_class = 10
 	max_combined_w_class = 35

```

Comment:
you can still hold the same amount of normal stuff, now you can optionally carry a mech thing too if you are so inclined